e/empty/empty_rhel_7.6.sh.txt

```diff
@@ -0,0 +1,46 @@
+# Version       : 0.10.1
+# Source        : https://github.com/iclanzan/empty.git
+# Tested on     : RHEL 8.3
+# Node Version  : v10.24.0
+# Maintainer    : Mohit Pawar <mohit.pawar@ibm.com>
+#
+# Disclaimer: This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+# ----------------------------------------------------------------------------
+
+set -e
+
+# Install all dependencies.
+yum clean all
+yum -y update

```

Comment:
if installation is required(i.e root), please mention that in script documentation